Source/Private/common.ps1

```diff
@@ -154,63 +143,18 @@ function _getApiVersion {
    [CmdletBinding(DefaultParameterSetName = 'Service')]
    param (
       [parameter(ParameterSetName = 'Service', Mandatory = $true, Position = 0)]
-      [ValidateSet('Build', 'Release', 'Core', 'Git', 'DistributedTask', 'VariableGroups', 'Tfvc', 'Packaging', 'MemberEntitlementManagement', 'ExtensionsManagement', 'ServiceFabricEndpoint', 'Graph', 'TaskGroups', 'Policy')]
+      [ValidateScript({$_ -in ([VSTeamVersions] | Get-Member -Static  -MemberType Property).name})]

```

Comment:
Versions has more values like Account, DefaultProject and Version that are not valid for Service

Reply:
Although they are not services, they can still be returned. If it's necessary to throw if someone uses _getApiVersion to find the account that can be done. 
However I am adding support for managing processes definitions which really needed its own API version  (possibly not with the most recent changes) and having to add to this list in common.ps1 when a change is made to class definition _felt_ wrong.